test/types/cube.jl

```diff
@@ -0,0 +1,129 @@
+@testset "cube" begin
+
+    x = reshape(1:24, 2, 3, 4)
+    @test is_transformable(x)
+
+    @testset "apply" begin
+
+        @testset "OneToOne" begin
+            T = FakeOneToOneTransform()
+            @test FeatureTransforms.apply(x, T) == ones(2, 3, 4)
+            @test FeatureTransforms.apply(x, T; inds=[1, 2]) == ones(2)
+            @test FeatureTransforms.apply(x, T; dims=2, inds=[2, 3]) == ones(2, 2, 4)
+        end
+
+        @testset "OneToMany" begin
+            T = FakeOneToManyTransform()
+            @test FeatureTransforms.apply(x, T) == ones(2, 6, 4)
+            @test FeatureTransforms.apply(x, T; inds=[1, 2]) == ones(2, 2)
+            @test FeatureTransforms.apply(x, T; dims=2, inds=[2, 3]) == ones(2, 4, 4)
+        end
+
+        @testset "ManyToOne" begin
+            T = FakeManyToOneTransform()
+            @test FeatureTransforms.apply(x, T; dims=1) == ones(3, 4)
+            @test FeatureTransforms.apply(x, T; dims=1, inds=[1, 2]) == ones(3, 4)
```

Comment:
Could test just one index here rather than 2, for something different, because the first dim only has size 2